specification/servicebus/data-plane/servicebus-swagger.json

```
@@ -1076,7 +1076,7 @@
           }
         },
         "supportOrdering": {
-          "description": "A value that indicates whether the topic supports ordering.",
+          "description": "Indicates if messages are received in the same order they are sent. If partitioned topics, defaults to false and setting it to true has no effect. For non-partitioned topics, setting it to false will improve perf, but messages may not received in the order they are sent.",
```

not "unpartitioned"?

```suggestion
          "description": "Indicates if messages are received in the same order they are sent. If partitioned topics, defaults to false, and setting it to true has no effect. For unpartitioned topics, setting it to false will improve perf, but messages may not be received in the order they are sent.",
```